llvm/lib/Target/Xtensa/XtensaMachineFunctionInfo.h

```diff
@@ -25,13 +25,14 @@ class XtensaMachineFunctionInfo : public MachineFunctionInfo {
   /// FrameIndex of the spill slot for the scratch register in BranchRelaxation.
   int BranchRelaxationScratchFrameIndex = -1;
   unsigned VarArgsFirstGPR;
-  int VarArgsStackOffset;
-  unsigned VarArgsFrameIndex;
+  unsigned VarArgsOnStackFrameIndex;
+  unsigned VarArgsInRegsFrameIndex;
```

Comment:
```suggestion
  int VarArgsInRegsFrameIndex;
```

Frame indexes are signed

Reply:
Fixed